debian/libze-intel-gpu1.symbols

```diff
@@ -554,7 +556,6 @@ libze_intel_gpu.so.1 libze-intel-gpu1 #MINVER#
  zetTracerExpSetEnabled@Base 24.35.30872.14
  zetTracerExpSetEpilogues@Base 24.35.30872.14
  zetTracerExpSetPrologues@Base 24.35.30872.14
- zexDeviceGetConcurrentMetricGroups@Base 24.35.30872.14
```

Comment:
this symbol has been renamed to zetIntelDeviceGetConcurrentMetricGroupsExp in the commit 6d86d562b596cab7194aa944b7b9267e2b716d0c , can we measure the impact of this symbol removal for API compability ? is this symbol  exposed as part of the API and used by applications ?

Reply:
even if it is used for a debug purpose, as soon as it is exposed as public API, if an existing application in Ubuntu uses this (since compute runtime is already in Ubuntu) and we get the new compute runtime in Ubuntu without this symbol (in the libze-intel-gpu1 with the same SONAME), the applications will break. i think it is ok for now and Timo will check it when we want to get into the Ubuntu archive. I will approve this change.

Reply:
yes please merge